src/panels/config/zha/zha-cluster-attributes.ts

```diff
@@ -0,0 +1,327 @@
+import {
+  html,
+  LitElement,
+  PropertyDeclarations,
+  PropertyValues,
+} from "@polymer/lit-element";
+import { TemplateResult } from "lit-html";
+import "@polymer/paper-button/paper-button";
+import "@polymer/paper-icon-button/paper-icon-button";
+import "@polymer/paper-card/paper-card";
+import "@polymer/iron-flex-layout/iron-flex-layout-classes";
+import "../../../components/buttons/ha-call-service-button";
+import "../../../components/ha-service-description";
+import "../ha-config-section";
+
+import { HomeAssistant } from "../../../types";
+import "../../../resources/ha-style";
+import { HassEntity } from "home-assistant-js-websocket";
```

Comment:
Can you move absolute imports (that don't start with a `.`) to the top?

Reply:
Should we add something like this to the suggested vscode extensions: https://marketplace.visualstudio.com/items?itemName=rbbit.typescript-hero